deployments/managed-services.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Managed services"
+---
+
+One of the key features of Architect.io is its dependency graph, which is a visual representation of the relationships between different services in an application.
+The dependency graph is a useful tool for managing complex applications with multiple services and dependencies, and demonstrates how those services work together in the application.
+
+Let's consider an example of a simple web application with a single `hello-world` service.
+The dependency graph for this application would have two nodes: one for the `hello-world` service and one for the Traefik `gateway`.
+
+![](/images/dependency-graph.png)
+
+# Gateway Node (traefik)
+The top node is a gateway node that uses Traefik to route traffic to the service. In this example, the gateway node would depend on the `hello-world` service, as it needs to route traffic to it.
+
+Architect uses Traefik as the reverse proxy to handle incoming traffic to all services. When a component is deployed, the graph automatically generates the gateway node.
```

Comment:
> Architect uses Traefik as the reverse proxy to handle incoming traffic to all services. When a component is deployed, the graph automatically generates the gateway node

This sort of makes it sound like we create Traefik when a deployment takes place that requires it, when in reality, it's already installed as a cluster application when the cluster is created. We could make it more clear in this doc the connection between cluster applications and a cluster's environments, why cluster applications are installed before environment deployments take place, and where the cluster applications live on the cluster